src/css/betterttv.css

```diff
@@ -2347,6 +2350,8 @@ body.channel_new.columns.ember-application {
     cursor: pointer;
 }
 
-.chat-line .bttv-deleted {
+.chat-line .bttv-deleted.bttv-click {
     opacity: 0.5;
+    text-decoration: none;
```

Comment:
`.bttv-deleted` is always on a deleted message, yes. I'm only applying the opacity to the `<message deleted>` text applied with the `.bttv-click` class. 

I believe for features like Show Delete Messages a deleted message shouldn't be opaque. It's open for personal preference.